package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.149.0",
   "description": "JavaScript 3D library",
   "type": "module",
-  "main": "./build/three.js",
```

Comment:
We could keep the `main` entry, but point it to the `.cjs` file:

```json
"main": "./build/three.cjs",
```

I'm not sure it matters, but could be better back-compat if the user is using (or compiling to) CommonJS module syntax in a bundler like Parcel that [doesn't support `exports` entries yet](https://github.com/parcel-bundler/parcel/issues/4155).

Reply:
Many projects are doing that with the main entry.  Some add an extra step to flip between prod and dev.

```json
"main": "./dist/index.js",
```

```js
// build/index.js
'use strict'

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./dist/app.prod.js')
} else {
  module.exports = require('./dist/app.cjs.js')
}
```

If that doesn't make sense, just collapse it down.

Reply:
@donmccurdy Updated! Hope it helps with back-compat 👍 .